### ISSUE TITLE
Blog: When visiting heading anchor, heading is barely visible

### ISSUE THREAD

**Issue by papojari** [aaranxu/adidoks#39]
When you visit the the anchor of a heading, the heading will be under the header and you will almost not see it.

Example from [here](https://papojari.codeberg.page/blog/home-manager-the-best-solution-to-your-dotfiles/#home-manager):
![image](https://user-images.githubusercontent.com/81317317/147865155-c9dd8607-de7b-45cd-a3ee-fd3a82b3c048.png)

**Comment by papojari** [comment]
Closing because it only seems to be an issue on my site

**Comment by papojari** [comment]
Wrong, it is an issue on the demo site too but just in the Blog section.

**Comment by ajyoon** [comment]
I've noticed this issue as well in certain contexts. It seems like Adidoks implements a workaround [in the docs context here](https://github.com/aaranxu/adidoks/blob/906865945f81a9c23b2a69ceb3ddd88e156a48d5/sass/layouts/_pages.scss#L1-L8), but that only applies to direct children of `.docs-content`, so it makes sense it wouldn't work in blogs.
